yaml/xyz/openbmc_project/Provisioning/Provisioning.interface.yaml

```diff
@@ -28,6 +28,8 @@ methods:
             type: string
             description: >
                 Identifier of the peer BMC to be provisioned.
+      errors:
+          - xyz.openbmc_project.Common.Error.InsufficientPermission
```

Comment:
please see if this error suits here
xyz.openbmc_project.Common.Error.NotAllowed?